### PR TITLE
Tweaked descriptions – Part 6

### PR DESCRIPTION
**Pull request by 4Luke4**

### CODE REVIEW

**Comment by lynxlynxlynx** [comment]
Looks fine, thanks.

**Comment by 4Luke4** [comment]
@lynxlynxlynx 

What about splitting `parameter2` in two halves if needed (f.i. op12, op9, etc.)? Something like:

```
n: 12
name: "HP: Damage"
param1: "Damage Amount"
param2a: "Mode"
param2b: "Damage Type"
savingthrow: "Save Type"
special: "Flags"
```

File: _opcodes/op109-bgee.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 n: 109
-name: "State: Hold"
+name: "State: Paralyze"
```
Comment (lynxlynxlynx) [inline _opcodes/op109-bgee.html:3]: Why the name change? Does at least NI also call it this way?

**Comment by lynxlynxlynx** [comment]
Re Param2: I don't think that would add much, since an explanation would be still required.

File: _opcodes/op109-bgee.html (hunk 1 above)
Comment (4Luke4) [reply _opcodes/op109-bgee.html:3]: Yes, NI calls it _Paralyze_ (guess to differentiate it from op175)

**Comment by 4Luke4** [comment]
> Re Param2: I don't think that would add much, since an explanation would be still required.

What about `0 + (256 << 16)` instead of `16777216`? That should be easier to read I think...

**Comment by lynxlynxlynx** [comment]
Maybe for us geeks, but it still requires a calculator that and tools use the final number, so I definitely don't see it as an improvement.

**Comment by 4Luke4** [comment]
@lynxlynxlynx 

Shouldn't Percentage Modifier be defined as `New = ('Statistic Modifier' value) / 100` instead of `New = (Current * 'Statistic Modifier' value) / 100`?

**Comment by lynxlynxlynx** [comment]
No, that would then be the same as the Set variant, but setting to 0-100, easily trying to set invalid values for some stats. It's a relative modifier.

**Comment by 4Luke4** [comment]
> No, that would then be the same as the Set variant, but setting to 0-100, easily trying to set invalid values for some stats. It's a relative modifier.

But it's not clear from the formula where's the difference between `p2=2` and `p2=5` (see `op126/176`...)

**Comment by lynxlynxlynx** [comment]
Because it is different only with several applications. You can add (non-stacking) to the mode name, I guess.

**Comment by 4Luke4** [comment]
> You can add (non-stacking) to the mode name, I guess.

Guess I'll do it at least for those two opcodes...

**Comment by 4Luke4** [comment]
@lynxlynxlynx 

Feel free to start reviewing...

Besides formatting, new tweaks include: op42/62, 43, 45, 46, 47/116, 53, 58, 60.

File: _opcodes/op262.html [hunk 2]
```diff
@@ -12,16 +12,17 @@
 ---
 Applies the visual range effect to the targeted creature(s).
 <br/>This effect modifies the visual range of the targeted creature(s).
-<br/>The Visual Range parameter seems to be in the 0-15 interval.
-<br/>Known values for 'Type' are:
-<br/>0   Cumulative Modifier -> Visual Range = Visual Range + 'Visual Range' value
-<br/>1   Flat Value Modifier -> Visual Range = 'Visual Range' value
-<br/>
+<br/>The Visual Range parameter seems to be in the <code>0–15</code> interval.
```
Comment (lynxlynxlynx) [inline _opcodes/op262.html:15]: If you're renaming the param, also change the users.

**Comment by lynxlynxlynx** [comment]
The rest looks fine, very nice!

**Comment by lynxlynxlynx** [comment]
Thanks!

**Pull request merged**